### ISSUE TITLE
Transition for open and close cookie options

### ISSUE THREAD

**Issue by Gismo1337** [Gismo1337/vue-cookie-consent-banner#20]
## **Is your feature request related to a problem? Please describe.**
When you click at options the checkboxes appear using v-if. Please add a smooth opening and closing transition.

*

---

## **Describe the solution you'd like**
Klick in options
Checkboxes appear with a smooth animation.

*

---

## **Describe alternatives you've considered**

*

---

### **Additional context**

*

**Comment by Gismo1337** [comment]
@alkhimch here

**Comment by alkhimch** [comment]
@Gismo1337  thank you 🙏

**Comment by alkhimch** [comment]
@Gismo1337  Can i create new checkbox component ?

**Comment by Gismo1337** [comment]
yes. but please try to keep it small and simple. no oversize and no more dependencies.

**Comment by alkhimch** [comment]
@Gismo1337 [https://github.com/Gismo1337/vue-cookie-consent-banner/pull/27](url)

**Comment by Gismo1337** [comment]
@alkhimch merged. nice feature. but the root of this issue was to make a transition for the checkbox overall. if you click @ options, the checkboxes appear. they could appear with a smooth transition =)

**Comment by Gismo1337** [comment]
@alkhimch  do you working on this?

**Comment by Gismo1337** [comment]
After adding the checkbox description, we should add a "opening animation"
![grafik](https://user-images.githubusercontent.com/6397577/193864844-fe479467-860e-4215-b55e-7294ca7f8dc1.png)

**Comment by alexjharrison** [comment]
Hey @Gismo1337 

I'll take this if no one's on it. Thanks!

**Comment by Gismo1337** [comment]
@alexjharrison assigned to you. have fun with this!

**Comment by alexjharrison** [comment]
Hey @Gismo1337

I couldn't get it to work among all screen sizes consistently.  You can unassign me and have someone else try it out.
